spec/linter-php-spec.js

```diff
@@ -69,4 +70,15 @@ describe('The php -l provider for Linter', () => {
     expect(messages[0].filePath).toBe(fatalPath);
     expect(messages[0].range).toEqual([[10, 4], [10, 25]]);
   });
+
+  it('handles deprecated errors', async () => {
+    const editor = await atom.workspace.open(deprecatedPath);
+    const messages = await lint(editor);
+
+    expect(messages[0].type).toBe('Warning');
+    // PHP 7.x
```

Comment:
You'll need to check what version of PHP is running in the CI environment (maybe there is an environment variable?) and test appropriately as the CI environment checks both 5.x and 7.x versions. You might need to enhance the Travis CI script with a variable of your own to do this if there isn't a default one.

(Apparently missed adding this to the previous review)
It should default to 7.x for when the tests are ran manually outside of the CI environment.

Reply:
Thank you for reviewing. I have been looking into ways to check the version of php without having to update the CI environment with something hard-coded. I will update once I get this working.